src/Illuminate/Container/Container.php

```diff
@@ -649,11 +649,9 @@ protected function resolve($abstract, $parameters = [])
         // We're ready to instantiate an instance of the concrete type registered for
         // the binding. This will instantiate the types, as well as resolve any of
         // its "nested" dependencies recursively until all have gotten resolved.
-        if ($this->isBuildable($concrete, $abstract)) {
-            $object = $this->build($concrete);
-        } else {
-            $object = $this->make($concrete);
-        }
+        $object = $this->isBuildable($concrete, $abstract)
```

Comment:
How is this any better than the IFs? I find `if(){ ... }else{ ... }` much clearer.